### ISSUE TITLE
Start in Fullscreen if game controller is present

### ISSUE THREAD

**Issue by Bluscream** [JosefNemec/Playnite#2025]
**Describe the solution you'd like**
A extra in checkbox in the settings that is parented by the "Start in Fullscreen" checkbox and forces Playnite to check if any controller is connected and not in standby and do the startup based on that

**Screenshots**
![](https://i.imgur.com/DkoaJSql.jpg)

**Comment by JosefNemec** [comment]
What does "active" mean? Just connected?

**Comment by Bluscream** [comment]
it means "not in any kind of standby mode" maybe some controllers report when they waited for input too long? Not sure

**Comment by JosefNemec** [comment]
That's not possible, we can either detect if a controller is connected or not.

**Comment by Bluscream** [comment]
that's fine with me aslong as the SDKs don't provide that

**Comment by Bluscream** [comment]
In addition to that it could be helpful if all starts of steam are overriden with starting steam Big Screen when a controller is connected because the normal UI isn't controller compatible

**Comment by slst19** [comment]
Is there any commandline way to open the FullScreen mode when an instance is already running in background.

**Comment by JosefNemec** [comment]
Yes, `--startfullscreen` but only works in [Playnite 8](https://playnite.link/docs/playnite8/manual/cmdlineArguments.html).
